packages/itwin/one-click-lca-widget/src/components/Reports.tsx

```diff
@@ -0,0 +1,146 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from "react";
+import { useMemo, useState, useEffect } from "react";
+import { SearchBox } from "@itwin/core-react";
+import { IModelApp } from "@itwin/core-frontend";
+import { useActiveIModelConnection } from "@itwin/appui-react";
+import { Button, Table } from "@itwin/itwinui-react";
```

Comment:
if this is the same generated reporting client used in the grouping-mapping widget, then you guys need to increase the priority of getting a standalong client out

Reply:
We have created insights client for all Insights and Reporting requests